sdk/eas/examples/simple-eas-workflow.ts

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/env bun
```

Comment:
Would remove this, idea is that all of this runs in any environment (node, bun...)